web/html/src/manager/virtualization/guests/edit/guests-edit.js

```diff
@@ -58,4 +60,6 @@ GuestsEdit.propTypes = {
     guest: PropTypes.object.isRequired,
 };
 
-export default GuestsEdit;
+module.exports = {
+  GuestsEdit: GuestsEdit,
```

Comment:
For future reference you can have only { GuestEdit }

https://alligator.io/js/object-property-shorthand-es6/

Reply:
I saw that with eslint :)